### PR TITLE
Add label element mixins.

### PR DESCRIPTION
**Pull request by elioschmutz**
Add new mixins for labels:

bildschirmfoto 2016-02-12 um 12 17 03

### CODE REVIEW

File: ftw/theming/resources/scss/elements/labels.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@mixin label($color-label: $color-primary) {
+  @include auto-text-color($color-label);
+  font-size: $font-size-base;
```
Comment (bierik) [inline ftw/theming/resources/scss/elements/labels.scss]: No need to define font-size explicitly.

File: ftw/theming/resources.zcml [hunk 2]
```diff
@@ -18,6 +18,7 @@
     <theme:scss file="resources/scss/globals/grid.scss" slot="mixins" />
     <theme:scss file="resources/scss/elements/button.scss" slot="mixins" />
     <theme:scss file="resources/scss/elements/list.scss" slot="mixins" />
+    <theme:scss file="resources/scss/elements/labels.scss" slot="mixins" />
```
Comment (bierik) [inline ftw/theming/resources.zcml]: Detail but use singular for file name. Have a look at the other elements.

**Comment by bierik** [comment]
Look great thx a lot :sparkles:

**Comment by elioschmutz** [comment]
@bierik kannst du das bitte bei Gelegenheit noch mergen?

**Pull request merged**